businessCentral/permissions/ADLSETrackDelete.PermissionSet.al

```diff
@@ -1,15 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-permissionset 82562 "ADLSE - Track Delete"
```

Comment:
Shouldn't we put this first on obsolete and remove it later?

Reply:
You're right! The upgrade path is more smooth and robust if we don't skip this.